src/ImageSharp/Formats/Tiff/Ifd/EntryReader.cs

```diff
@@ -116,6 +116,15 @@ private static bool GetIsArray(ExifTagValue tagId, uint count)
 
         private bool ReadValueOrOffset(ExifValue entry, ExifDataType rawDataType, uint count)
         {
+            if (entry.Tag == ExifTag.SubIFDOffset
+                || entry.Tag == ExifTag.GPSIFDOffset
+                /*|| entry.Tag == ExifTagValue.SubIFDs*/)
+            {
+                // todo: ignore subIfds (exif, gps)
+                this.stream.Skip(4);
```

Comment:
why is that skipped here, isn't that needed later on for reading exif?

Reply:
here is skipping sub Ifd tags (sub exif and gps profiles), which are not yet supported for reading (here only their address in the stream), these tags were removed so as not to be misleading